changelogs/fragments/530-parse_cli.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - parse_cli: add support for multiple matches inside a block by adding new dictionary key to result
```

Having a colon breaks yaml parsing. You can quote the whole item to work around it but by convention we usually use another dash
```suggestion
  - parse_cli - add support for multiple matches inside a block by adding new dictionary key to result
```